### DISCUSSION TITLE
Need to move one of my files to the tools directory

### DISCUSSION THREAD

**Discussion by verquepasa** [devlooped/nugetizer#103] [Q&A] [answered]
First: thanks for this tool!

I'm trying to put a file that is in /bin/debug/file.dacpac into the /tools directory, as this is a requirement for a specific type of build described in msbuild.sdk.sqlproj project

``
    <PackageFile Include="**\bin\**\*.dacpac" PackagePath="tools\"   />
``

The problem is the file is copied with the whole relative dir
I want:
``
 /tools/file.dacpac
``
and I get:
``
 /tools/bin/Debug/net5.0/file.dacpac
``

Is there a way I can remove the _RelativeDir_  ?

**Comment by kzu** [comment, marked as answer]
Sure thing, just build the full path you want to have:

```xml
<PackageFile Include="**\bin\**\*.dacpac" PackagePath="tools\%(RelativeDir)%(Filename)%(Extension)" />
```

Note that since you're using double `**` I'm not sure what the `%(RelativeDir)` metadata will be, but you can build the actual final location for every item you want to include. This is plain MSBuild syntax 👍

**Reply by verquepasa** [reply]
Thanks!, that worked
